mkdocs.yml

```diff
@@ -1,6 +1,7 @@
 site_name: 'Pi-hole documentation'
 site_url: 'https://docs.pi-hole.net/'
-repo_url: 'https://github.com/pi-hole/docs'
+repo_url: 'https://github.com/pi-hole/pi-hole'
+edit_uri: '../docs/blob/master/docs/'
```

Comment:
Do note that I didn't use `/edit/` here so that it doesn't require a login to view the file.